packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -269,6 +282,7 @@ export class BasePicker<T extends {}, P extends IBasePickerProps<T>>
 
     const suggestionsVisible = !!this.state.suggestionsVisible;
     const suggestionsAvailable = suggestionsVisible ? this._ariaMap.suggestionList : undefined;
+    const hasError = typeof (this.state.errorMessage || this.props.errorMessage) !== 'undefined';
```

Comment:
I think it's probably better if you define this as you did below:

```suggestion
    const hasError = !!(this.state.errorMessage ?? this.props.errorMessage);
```